config/nginx.conf

```
@@ -56,6 +56,10 @@ http {
     ssl_ciphers 'EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH';
     ssl_dhparam /etc/nginx/ssl/dhparam.pem;
 
+    location ~* /\.git\/.* {
```

```suggestion
    location ~ /\. {
```
Might be safer to block all dot dirs / dotfiles.

---

Yep definitely